src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/NoJSButton.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
```

Comment:
This component is getting more and more complex as we try to make it support two different use cases, the veggie and non veggie buttons. I think we might have got to the point where it makes sense to split it out into two, what do you think? There would be some code duplication but it would create two much simpler components that are easier to test and easier to maintain and reason about.

Reply:
I am happy to move the CSS back to their respective sections, I still think the `NoJSButton` should be its own component however.

Reply:
I removed this component in the end, moving the logic to VeggieBurger and ExpandedMenueToggle